### ISSUE TITLE
Add command line support

### ISSUE THREAD

**Issue by mindspank** [mindspank/buildapp#1]
Add command line supports, would have to introduce dependency on qsocks. Worth it?

**Comment by chrislarsenqlik** [comment]
Yes, this would be good to simply test qsocks (/functions) without having to generate node code.
